### DISCUSSION TITLE
Which docker tag use for Ada Lovelace cards?

### DISCUSSION THREAD

**Discussion by xfalcox** [huggingface/text-embeddings-inference#13] [Q&A] [answered]
For developers with RTX 4090 and similar cards, which docker tag should we use to run ?

Currently the tags on the README are:

Architecture | Image
-- | --
CPU | ghcr.io/huggingface/text-embeddings-inference:cpu-latest
Turing | ghcr.io/huggingface/text-embeddings-inference:turing-latest
Ampere 80 | ghcr.io/huggingface/text-embeddings-inference:latest
Ampere 86 | ghcr.io/huggingface/text-embeddings-inference:86-latest
Hopper | ghcr.io/huggingface/text-embeddings-inference:hopper-latest

**Comment by OlivierDehaene** [comment, marked as answer]
Hello!
I just added a new `89-latest` target for Ada Lovelace. You should also be able to use the `latest` and `86-latest`.
